torch/csrc/autograd/python_variable.cpp

```diff
@@ -414,7 +416,12 @@ static PyObject* THPVariable_make_wrapper_subclass(PyObject*, PyObject* args, Py
   // to continue on to the underlying CPU/CUDA kernel advertised by the dispatch
   // key, which will immediately segfault because the data pointer is null.  By
   // forcing users to define __torch_dispatch__ we ensure this does not happen
-  TORCH_CHECK_TYPE(PyObject_FastGetAttrString(cls, "__torch_dispatch__").ptr() != nullptr,
+  // TODO: This check is not complete; because the user can disable torch
+  // dispatch and then go again, triggering segfault.  TBH I'm thinking I want
+  // to delete this function entirely
```

Comment:
You want to delete _make_wrapper_subclass?

Reply:
Well, I can't, because it does legitimately solve a problem that cannot be solved any other way. But it feels like there should be a better way of doing this.